docs/SUMMARY.md

```diff
@@ -4,6 +4,7 @@
 * [Live Editor](https://semgrep.live)
 * [Pattern Features](pattern-features.md)
 * [Configuration Files](configuration-files.md)
+* [Output](output.md)
 * [Languages](matrix.md)
 * [Integrations](integrations.md)
 * [Development](development.md)
```

Comment:
@DrewDennison / @mschwager  -- how are people finding this summary.md doc anyways? Should we link to the output.md from the main readme?

Reply:
Good question - I think we need to have a broader discussion on where we should be directing people for documentation. Are we set on GitBook? Is Github + Markdown sufficient? Something else? Currently we direct people to the Github + Markdown docs, but the GitBook is also available: https://docs.semgrep.dev/. I've mostly been creating content and delaying that decision - maybe @pabloest has thoughts too?

Reply:
I have not looked into writing/publishing with GitBook, but as a reader I find it very pleasing to consume and digest. That may be just the theme @mschwager chose and something similar would be possible using Readthedocs. 

For writing and deploying docs I found Readthedocs a bit clunky but I only used it a few times.

In the short term we should definitely link to the docs and in the longer term we could consider integrating GitBook (or whatever we choose) into our website.

Reply:
@DrewDennison pulled in GitBook initially. I'm not opposed to it, but I don't generally think we have a cohesive documentation story at the moment. At some point we'll probably want to be consistent with things like Bento and other projects :+1: